setup.cfg

```diff
@@ -2,5 +2,5 @@
 test=pytest
 
 [tool:pytest]
-addopts=--cov=tf2onnx --ignore=tests/test_custom_rnncell.py --ignore=tests/test_const_fold.py
+addopts=--cov=tf2onnx --ignore=tests/test_custom_rnncell.py --ignore=tests/test_const_fold.py --ignore=tests/test_optimizers.py
```

Comment:
consider something like unittest.skipIf(onnxruntime.__version__ == 0. 1.3) ?

Reply:
0.1.4 seems to be ready soon, I will enable it after your pr is merged.